### DISCUSSION TITLE
Rips got inf value

### DISCUSSION THREAD

**Discussion by Darya1998** [scikit-tda/ripser.py#169] [Q&A]
Hello!
operation rips.fit_transform(data) gives me INF value in the last row of first distribution. Could I substitute is with a big value or it is stupid? Will I loose something important? After this operation I want to use persim.sliced_wasserstein, as a result I get NAN value, it is because of INF value.
Thank you in advance!

**Comment by catanzaromj** [comment]
@Darya1998 Assuming you computed persistent homology in degree 0, this is totally expected. As long as your point cloud has at least one point in it, the last death time will always be infinity. This corresponds to the fact that the point cloud has a point in it. So you lose no information by disregarding it.
